src/tests/ftest/container/boundary.yaml

```diff
@@ -0,0 +1,37 @@
+hosts:
+  test_servers:
+    - server-A
+    - server-B
+    - server-C
+    - server-D
+timeout: 3500
```

Comment:
Actual test time was at most 12 minutes: https://build.hpdd.intel.com/blue/organizations/jenkins/daos-stack%2Fdaos/detail/PR-8605/6/tests/

Is this large timeout to account for pool destroy variability?
FWIW, @phender is working on a fix for this

Reply:
Have updated proper value for the timeout